### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/code/kotlin-playground.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#972] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/code/kotlin-playground.html

使用 为了使用 kotlin 交互演示，你应该使用一个名为 kotlin-playground 的容器。 在其中，你可以使用 2 个指令: @file 文件名 紧跟文件的代码块 第一个文件将是主文件，其他文件将被视为隐藏的依赖项。 @setting 紧跟一个自定义设置的 json 块 支持的选项: data-version: 目标 Kotlin 编译器...

https://theme-hope.vuejs.press/zh/guide/markdown/code/kotlin-playground.html

**Comment by yuanyuyu1123** [comment]
假如要支持rust该怎么做呢?

**Reply by Mister-Hope** [reply]
首先你要找到一个可以在js上运行的rust runtime，或者一个确定欢迎任意第三方调用的 rust playground api。

注：随意调用他人在自己网站暴露的API在全球大部分地区都有极大的法律风险
